### ISSUE TITLE
Enhancement request: convert some modal dialogs to normal windows

### ISSUE THREAD

**Issue by rpgoldman** [protegeproject/protege#1031]
I was trying to debug a problem with the web catalog, where my imports were not being found as I expected they should.

This was made more difficult by the fact that both the XML catalog editor window and the Log transcript window are modal dialog boxes.

I don't *think* this should be necessary.  Any chance they could be converted to normal windows?

**Comment by rpgoldman** [comment]
One more is the "Ontology source locations" window. Again, there doesn't seem any reason this should be a modal dialog.

**Comment by ykazakov** [comment]
I do not think that the log transcript window (Window > Show Log...) is a modal dialog. Or do you mean some other log window?
By "XML catalog editor" do you mean "File > Edit ontology catalog file..."? It looks like just a file picker to me.
I guess by "Ontology source locations" you mean "File > Loaded ontology sources..." Right?

**Comment by rpgoldman** [comment]
Yes, that’s right.  Perhaps “modal dialog” is the wrong term, but when one of these windows is open, it sits in front and I cannot interact with other Protege windows until I close them.

**Comment by ykazakov** [comment]
Correct, that is what modal dialog usually means. However, with the log window opened, I can interact with Protegé just fine. So the log window is not modal.

Also, I do not think it is possible to change the file picker window not to be modal. In other programs (e.g. open a file in word) it is the same.
